app/controllers/rooms_controller.rb

```diff
@@ -180,6 +180,7 @@ def start
     opts[:mute_on_start] = room_setting_with_config("muteOnStart")
     opts[:require_moderator_approval] = room_setting_with_config("requireModeratorApproval")
     opts[:record] = record_meeting
+    opts[:avatarURL] = current_user.image
 
```

Comment:
@farhatahmad , do we need to check here if current_user.image is present? It seems unnecessary to me.